script.js

```diff
@@ -1,4 +1,569 @@
+  return outputValue;
+};
+
+// Create a helper function to display current players cards and available options
+var displayPlayer = function () {
+  var name = playerStatus[currentPlayer - 1].name;
+  var hands = playerStatus[currentPlayer - 1].hand;
+  var bet = playerStatus[currentPlayer - 1].bet;
+  var credit = playerStatus[currentPlayer - 1].credits;
+
+  var outputValue = `${name}, its your turn! You have placed a bet of ${bet}, with a remaining credit of $${credit}. Your hand is displayed below.`;
+
+  for (handsIndex = 0; handsIndex < hands.length; handsIndex += 1) {
+    outputValue += `<br><b>Hand ${handsIndex + 1}:</b>`;
+    for (handIndex = 0; handIndex < hands[handsIndex].length; handIndex += 1) {
+      var card = hands[handsIndex][handIndex];
+      outputValue += ` ${card.name} of ${card.suit}${card.suitIcon} |`;
+    }
+  }
+
+  // Check for potential blackjack condition and/or display available actions to player
+  if (hands.length == 1) {
+    if (checkBlackJack(hands[0]) == true) {
+      outputValue += `Congratulations, you have hit blackjack! 🤑🤑🤑<br><br>`;
+      playerStatus[currentPlayer - 1].blackjack = true;
+      playerStatus[currentPlayer - 1].score[0] = 21;
+
+      updatePlayer();
+      if (gameState == 3) {
+        var playerName = playerStatus[currentPlayer - 1].name;
+        outputValue += `${playerName} is next up. Click "Submit" to see your cards and decide on your options.`;
+      } else {
+        outputValue += `This is the end of the player phase of the game. Click "Submit" to see the outcome of the dealers turn.`;
+      }
+      return outputValue;
+    }
+  }
+
+  outputValue += `<br><br>You may input your desired action in the input field and click submit. Valid inputs are as follows:<br>
+  "Hit" - Draw a card for ALL hands.<br>
+  "Hit N" - Draw a card for hand N.<br>
+  "Stand" - End your turn once you are satisfied with your card.`;
+
+  if (splitEligibility() == true) {
+    outputValue += `"Split" - Split your current identical cards into two seperate hands. This will draw one card for each hand and apply your original bet to each hand.`;
+  }
+
+  return outputValue;
+};
+
+// Create a helper function for player actions
+var playerState = function (input = "") {
+  console.log(input);
+  var outputValue = `Sorry your input is invalid. `;
+
+  if (input == "") {
+    return displayPlayer();
+  } else if (input == "Stand") {
+    return standPlayer();
+  } else if (input.includes("Hit")) {
+    var inputList = input.split(" ");
+    if (inputList.length == 2) {
+      if (Number.isInteger(Number(inputList[1]))) {
+        return hitPlayer(Number(inputList[1]));
+      }
+    } else if (inputList.length == 1 && inputList[0] == "Hit") {
+      return hitPlayer();
+    }
+  } else if (input == "Split") {
+    if (splitEligibility() == true) {
+      return splitPlayer();
+    } else {
+      outputValue += "You are unable to split with your current hand.";
+      return outputValue;
+    }
+  }
```

Comment:
if a function has many ifs, how many ifs must there be if we were to say that there are too many ifs? Maybe we should 'switch' the approach

Reply:
here's a link to the switch statement that you can use https://www.w3schools.com/js/js_switch.asp

Reply:
Thanks! Didn't know this elegant method exists!